data/types/common/duration.go

```diff
@@ -1,4 +1,4 @@
-package physical
+package commontypes
```

Comment:
As a golang convention, package names should reflect the base name of the directory
```suggestion
package common
```

Reply:
I reviewed all the package names.